app/models/users/user.py

```
@@ -4,40 +4,54 @@
 
 import enum
 import uuid
+from typing import TYPE_CHECKING
 
 from datetime import datetime
 
-from sqlalchemy import Boolean, Column, DateTime, Integer, String, func
+from sqlalchemy import Boolean, Column, DateTime, String, func, Enum as SQLAlchemyEnum
```

Ruff (0.12.2)

`sqlalchemy.Column` imported but unused

Remove unused import: `sqlalchemy.Column`

(F401)